contracts/mainnet/connectors/pooltogether/main.sol

```diff
@@ -0,0 +1,203 @@
+     * @notice Deposit assets into the Prize Pool in exchange for tokens
+     * @param prizePool PrizePool address to deposit to
+     * @param to The address receiving the newly minted tokens
+     * @param amount The amount of the underlying asset the user wishes to deposit. The Prize Pool contract should have been pre-approved by the caller to transfer the underlying ERC20 tokens.
+     * @param controlledToken The address of the token that they wish to mint. For our default Prize Strategy this will either be the Ticket address or the Sponsorship address.  Those addresses can be looked up on the Prize Strategy.
+     * @param referrer The address that should receive referral awards, if any.
+     * @param getId Get token amount at this ID from `InstaMemory` Contract.
+     * @param setId Set token amount at this ID in `InstaMemory` Contract.
+    */
+
+    function depositTo(
+        address prizePool,
+        address to,
+        uint256 amount,
+        address controlledToken,
+        address referrer,
+        uint256 getId,
+        uint256 setId
+    ) external payable returns ( string memory _eventName, bytes memory _eventParam) {
+        uint _amount = getUint(getId, amount);
+
+        PrizePoolInterface prizePoolContract = PrizePoolInterface(prizePool);
+        address prizePoolToken = prizePoolContract.token();
+
+        // Approve prizePool
+        TokenInterface tokenContract = TokenInterface(prizePoolToken);
+        tokenContract.approve(prizePool, _amount);
+
+        prizePoolContract.depositTo(to, _amount, controlledToken, referrer);
+
+        setUint(setId, _amount);
+
+        _eventName = "LogDepositTo(address,address,uint256,address,address,uint256, uint256)";
+        _eventParam = abi.encode(address(prizePool), address(to), _amount, address(controlledToken), address(referrer), getId, setId);
+    }
+
+    /**
+     * @dev Withdraw from Prize Pool
+     * @notice Withdraw assets from the Prize Pool instantly.  A fairness fee may be charged for an early exit.
+     * @param prizePool PrizePool address to deposit to
+     * @param from The address to redeem tokens from.. This means you can withdraw on another user's behalf if you have an allowance for the controlled token.
+     * @param amount The amount of tokens to redeem for assets.
+     * @param controlledToken The address of the token to redeem (i.e. ticket or sponsorship)
+     * @param maximumExitFee The maximum early exit fee the caller is willing to pay. This prevents the Prize Strategy from changing the fee on the fly.  This should be pre-calculated by the calculateExitFee() fxn.
+     * @param getId Get token amount at this ID from `InstaMemory` Contract.
+     * @param setId Set token amount at this ID in `InstaMemory` Contract.
+    */
+
+    function withdrawInstantlyFrom (
+        address prizePool,
+        address from,
+        uint256 amount,
+        address controlledToken,
+        uint256 maximumExitFee,
```

Comment:
Any example value, how `maximumExitFee` looks like?

Reply:
I've added code of how it can be calculated in the test script, Something like this:

` let prizePoolContract = new ethers.Contract(DAI_PRIZE_POOL_ADDR, prizePoolABI, ethers.provider);`
`let earlyExitFee = await prizePoolContract.callStatic["calculateEarlyExitFee"](dsaWallet0.address, PT_DAI_TICKET_ADDR, amount);`

https://github.com/eccheung4/dsa-connectors/blob/pooltogether-connector/test/pooltogether/pooltogether.test.js#L170-L178